gradle.properties

```diff
@@ -1,4 +1,5 @@
 android.enableJetifier=true
 android.useAndroidX=true
 kotlin.code.style=official
-android.enableUnitTestBinaryResources=true
\ No newline at end of file
+android.enableUnitTestBinaryResources=true
+org.gradle.parallel=true
```

Comment:
Sounds good but isn't it defeated by the line I first commented?

Reply:
From what I understood: 

`org.gradle.parallel` allows parallelism for `build`

Reply:
https://guides.gradle.org/performance/#parallel_execution